### ISSUE TITLE
Alternative computation for radiant temperature

### ISSUE THREAD

**Issue by Mathadon** [open-ideas/IDEAS#623]
EnergyPlus uses a different way of calculating the equivalent radiant temperature of the environment.

Currently we use:

```
final parameter Real Fssky=(1 + cos(inc))/2
    "radiant-interchange configuration factor between surface and sky";
```

edit:
but energy plus uses

```
final parameter Real Fssky=(1 + cos(inc))/2*sqrt((1 + cos(inc))/2)
    "radiant-interchange configuration factor between surface and sky";
```

They reference  `McClellan, T. M., & Pedersen, C. O. (1997). Investigation of Outside Heat Balance Models for Use in a Heat Balance Cooling Load Calculation Procedure. ASHRAE Transactions, 103, 469–484.`for this equation, but this source suggests:

```
final parameter Real Fssky=(1 + cos(inc))/2*cos(inc/2))
    "radiant-interchange configuration factor between surface and sky";
```

The last two formulations however seem to produce the same numerical result.

This produces clear differences in the `ZoneExample` model:
![screen shot 2017-01-11 at 11 57 55](https://cloud.githubusercontent.com/assets/5970879/21846015/43c5b1a8-d7f5-11e6-81ba-3dc94d1c84d5.png)

I propose to change the equation to the second formulation since it seem to be more appropriate and better supported by literature. @rubenbaetens is there a reason not to do this?

edit2: 
According to https://en.wikipedia.org/wiki/List_of_trigonometric_identities both formulations are mathematically identical (see Half-angle formulae).

**Comment by GlennReynders** [comment]
I don't really see where this `*cos(inc/2))` should come from.
The angle factor for the sky is given by: 
![image](https://cloud.githubusercontent.com/assets/4417460/21895959/e5c95e7e-d8e4-11e6-9427-664d2aea6d27.png)
Integration over half of the "sky dome" results in: 
![image](https://cloud.githubusercontent.com/assets/4417460/21895999/0b9e3dcc-d8e5-11e6-8b0e-8ac6cf836fde.png)

So, as far as I'm aware no need to change the current model.

**Comment by Mathadon** [comment]
The view factor is indeed correct. The problem is apparently that what you 'see' is not the black sky temperature, but a weighted sum of the black sky temperature and the air temperature since the air radiates as well, especially when the surface is vertical since then the path through the air is longer towards space.

According to`Walton, G. N. 1983. Thermal Analysis Research Program Reference Manual. NBSSIR 83-2655. National Bureau of Standards,`
"The effective temperature of any point in the sky depends on the optical path length through the atmosphere."

**Comment by GlennReynders** [comment]
solved in #647
